ste/sender-azureFile.go

```diff
@@ -42,6 +43,8 @@ type FileClientStub interface {
 	URL() string
 }
 
+var doWithOverrideReadOnly = common.DoWithOverrideReadOnlyOnAzureFiles
```

Comment:
why make a variable for the function instead of just calling the function directly?